source/api/content-state/0.2/index.md

```diff
@@ -0,0 +1,510 @@
+
+The value passed to a client via this parameter (or other supported mechanisms) may be the `target` property of an implied annotation:
+
+```json
+{"id":"https://example.org/iiif/item1/manifest","type":"Manifest"}
+```
+
+This form is better suited to scenarios where compactness is important, for example a query string parameter, where it is also more likely to be combined with Base64-encoding.
+
+The client _MUST_ examine the form of the string value, or the decoded string value, of the property to determine whether it is JSON-LD or a URL, and if it is JSON-LD the client _MUST_ check the `type` property to determine whether it is a full annotation or just the target content state.
+
+__Queries__<br/><br/>Does the inline full annotation version require `id`?<br/>Does it require `@context`?<br/>Should we require a different parameter name when the value is just the target of the annotation, rather than the full annotation, or should we stick to one parameter and require the client to check its `type` and process accordingly?<br/>For nicer query string parameters, should we permit a syntax that is invalid JSON-LD but valid JSON in the same form (i.e., no quotes, and/or single quotes), or even not valid JSON?:<br/><br/>`viewer.html?iiif-content={"id":"https://example.org/iiif/item1/manifest","type":"Manifest"}`<br/><br/><a href="http://tify.sub.uni-goettingen.de/demo.html?manifest=https://manifests.sub.uni-goettingen.de/iiif/presentation/PPN857449303/manifest">See TIFY to get an idea of what this feels like</a> (observe the `tify` parameter as you change the content state).<br/><br/>Should the client be required to accept the JSON-LD in base64-encoded form? [Yes!]
+{: .warning}
+
+## 3. Initialisation
+{: #initialisation}
+
+This specification provides mechanisms that IIIF compatible software can use to expose, share and transfer content state descriptions, but does not specify what form IIIF compatible software itself may take (e.g., a web page, a JavaScript web application, a native mobile application, a desktop application, or display kiosk hardware). Please see the [IIIF Cookbook][annex-cookbook] for further examples of the patterns listed below.
+
+TODO - provide cookbook links to fully working examples for all of these
+
+### 3.1. Initialisation mechanisms (protocol)
+{: #initialisation-mechanisms}
+
+The data structure _MAY_ be made available to the client via a number of methods, including (but not limited to):
+
+#### 3.1.1 Linking: HTTP GET (query string) parameter
+{: #initialisation-mechanisms-link}
+
+In the following examples, the same annotation is used each time. As JSON:
+
+```json
+{
+  "type": "Annotation",
+  "motivation": "highlighting",
```

Comment:
["highlighting"]  (as array)

Reply:
Don't think this change is necessary, but probably is still useful. The issue is #1764.